JavaScript/Topics/Data_Structures/README.md

```diff
@@ -0,0 +1,13 @@
+# Iterables, Sets and Maps in JavaScript objects
+
+## What is a MAP?
+
+ A JavaScript Map is an associative collection of distinct keys and values. Both keys and values can be any primitive or object. This is a significant data structure with many valuable uses.
+
+ ## What is set?
+
+ Set is another new collection introduced by ES6. The Array class of JavaScript can work with ordered collection of data, but not so well with unordered collections or when values held in the collection are unique. That’s why JavaScript introduced Set.
+
+  _a Map is a collection of ordered key/value pairs, and that a Set is a collection of unique values. Both of these data structures add additional capabilities to JavaScript and simplify common tasks such as finding the length of a key/value pair collection and removing duplicate items from a data set, respectively. On the other hand, Objects and Arrays have been traditionally used for data storage and manipulation in JavaScript, and have direct compatibility with JSON, which continues to make them the most essential data structures, especially for working with REST APIs. Maps and Sets are primarily useful as supporting data structures for Objects and Arrays._
```

Comment:
Map definition should be under `What is a MAP?` section

Reply:
Sir, it's a kind of conclusion of what the folder contains (line number 11).
Should I still change it?
@dakshsethi

Reply:
Ok then just give one more space and at least write the heading as Conclusion or something
Also mention `**Contributor:** [Sakshi Mishra](https://github.com/SakshiMishra1)` at the end of every page

Reply:
Ok sir, making the changes.

Reply:
Done with the changes sir. @dakshsethi